pr-triage-procedure.md

```diff
@@ -138,6 +146,21 @@ been more than 5 days since an update, ping the person starting the run on IRC a
 
 [S-waiting-on-crater PRs]: https://github.com/rust-lang/rust/pulls?utf8=%E2%9C%93&q=is%3Aopen%20is%3Apr%20sort%3Aupdated-asc%20label%3AS-waiting-on-crater
 
+### [S-blocked PRs][S-blocked]
+
+PRs inactive for greater than 7 days need to be processed. These can
+be found by looking at the "updated X days ago" on GitHub's PR list.
+
+Find the blocking issue from the triage comment and see if it has been
+resolved.
```

Comment:
Maybe we want to propose some uniform message so that we can easily find these?

> This PR appears to be blocked on #14545.

Something like that, maybe? Or just "blocked on #xxx" being standardized would probably be enough.

Reply:
> some uniform message

Added your example to the paragraphs describing adding the comment, thanks!